Code_parser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "Code_parser.h"
+
+void Code_parser::doParse(const string file_path) {
```

Comment:
It is better to redefine the first function argument as a reference. Consider replacing 'const .. file_path' with 'const .. &file_path'.

Reply:
Resolved